### PR TITLE
Feat-Responsiveness

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added Responsiveness

### CODE REVIEW

**Review by manojkumar8400 — APPROVED** [review]
your app is perfectly responsive, you did great work with your UI.

File: src/frontend/components/Sidebar/Sidebar.css [hunk 1]
```diff
@@ -20,16 +20,75 @@
   margin: 0.5rem 0;
   cursor: pointer;
   border-radius: 0.3rem;
+
 }
 .sidebar_wrapper > *:hover {
+  background-color: var(--bg-color);
+}
+.sidebar2 {
```
Comment (srishti-maurya) [inline src/frontend/components/Sidebar/Sidebar.css:28]: you may give it a better name maybe

File: src/frontend/components/Navbar/Navbar.css [hunk 2]
```diff
@@ -1,50 +1,61 @@
...
-    align-items: center;
-   
+.Navbar_lastitems {
+  display: flex;
+  align-items: center;
 }
-.light_mode-btn{
-  padding:0 1rem 0 0;
+.light_mode-btn {
+  padding: 0 1rem 0 0;
```
Comment (srishti-maurya) [inline src/frontend/components/Navbar/Navbar.css:37]: you may directly write it as `padding-right`

File: src/frontend/components/Navbar/Navbar.css [hunk 3]
```diff
@@ -1,50 +1,61 @@
...
-    box-sizing: border-box;
-    justify-content: space-between;
-    padding:0 2rem;
-    position: sticky;
-    left:0;
-    top:0;
-    z-index: 2;
-
+.Navbar_wrapper {
+  /* min-width: 100vw; */
```
Comment (srishti-maurya) [inline src/frontend/components/Navbar/Navbar.css:2]: you may remove the unused code

**Review by srishti-maurya — APPROVED** [review]
Looks good!
You may avoid directly styling the tags.

**Review by bipin7yadav — APPROVED** [review]
Ui looks good

**Pull request merged**